source/tests/system/nidigital_driver_api_tests.cpp

```diff
@@ -327,6 +327,51 @@ TEST_F(NiDigitalDriverApiTest, SelfCalibrate_CompletesSuccessfully)
   expect_api_success(response.status());
 }
 
+TEST_F(NiDigitalDriverApiTest, ClockGeneratorInitiate_ClockGenerationInitiated)
+{
+  std::string channel_list = "0";
+  ::grpc::ClientContext context;
+  digital::ClockGeneratorInitiateRequest request;
+  request.mutable_vi()->set_id(GetSessionId());
+  request.set_channel_list(channel_list);
+  digital::ClockGeneratorInitiateResponse response;
+  ::grpc::Status status = GetStub()->ClockGeneratorInitiate(&context, request, &response);
+
+  EXPECT_TRUE(status.ok());
+  expect_api_success(response.status());
+  EXPECT_TRUE(get_bool_attribute("0", digital::NIDIGITAL_ATTRIBUTE_CLOCK_GENERATOR_IS_RUNNING));
+}
+
+TEST_F(NiDigitalDriverApiTest, ConfigureSoftwareEdgeStartTrigger_SoftwareEdgeStartTriggerConfigured)
```

Comment:
Looking at the test code, it seems like this would be a more appropriate 2nd section: `..._StartTriggerTypeAttributeIsSoftware`

Reply:
changed it